src/service/mobile_config.proto

```diff
@@ -0,0 +1,129 @@
+syntax = "proto3";
+
+package helium.mobile_config;
+
+// ------------------------------------------------------------------
+// Message Definitions
+// ------------------------------------------------------------------
+
+// == Field Notes ==
+//
+// - Every message including a `signature` field will need to be signed over the
+//   complete message, with the `signature` field set to an empty value.
```

Comment:
Be clear about req vs res signing and who is doing the signing

Reply:
fixed [05ef4df](https://github.com/helium/proto/pull/297/commits/05ef4df63be35fb6a973182cf8641d2d0c8f8765)